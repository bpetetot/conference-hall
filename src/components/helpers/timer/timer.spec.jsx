/* eslint-env jest */
import snap from 'tests/snapshot'
import { mount } from 'enzyme'
import React from 'react'
import Timer from './index'

const snapshot = props => snap(Timer)({ ...props })

describe('components/timer', () => {
  const props = {
    delay: 20,
    enabled: true,
    children: 'timer',
    onFinish: () => {},
  }
  it('should render with children', snapshot(props))
  it('should start when enable', () => {
    const wrapper = mount(<Timer {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
