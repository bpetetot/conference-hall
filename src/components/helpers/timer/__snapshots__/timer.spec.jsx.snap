// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/timer should render with children 1`] = `"timer"`;

exports[`components/timer should start when enable 1`] = `
<Timer
  delay={20}
  enabled={true}
  onFinish={[Function]}
>
  timer
</Timer>
`;
